### PR TITLE
Pubsub to BigTable new template in Java

### PR DESCRIPTION
**Pull request by balajiss2**
#43 
This is a template for Pubsub to BigTable in Java

### CODE REVIEW

File: java/src/main/java/com/google/cloud/dataproc/templates/pubsub/PubSubToBigTable.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  }
+
+  @Override
+  public void runTemplate() {
+
+    validateInput();
+
+    JavaStreamingContext jsc = null;
+
+    try {
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/pubsub/PubSubToBigTable.java:73]: please remove try-catch

File: java/src/main/java/com/google/cloud/dataproc/templates/pubsub/PubSubToBigTable.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                        BigtableDataClient.create(
+                            pubSubBigTableOutputProjectId, pubSubBigTableOutputInstanceId);
+
+                    while (sparkPubsubMessageIterator.hasNext()) {
+                      SparkPubsubMessage message = sparkPubsubMessageIterator.next();
+
+                      // try {
+                      JSONObject record = new JSONObject(new String(message.getData()));
+                      long timestamp = System.currentTimeMillis() * 1000;
+                      try {
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/pubsub/PubSubToBigTable.java:139]: remove try-catch

File: java/src/main/resources/template.properties [hunk 3]
```diff
@@ -151,6 +151,14 @@ pubsub.bq.output.table=<bq output table>
 ## Number of records to be written per message to BigQuery
 pubsub.bq.batch.size=1000
 
+# PubSubToBigTable properties
+## reuse pubsub.input.project.id, pubsub.input.subscription, pubsub.timeout.ms, pubsub.streaming.duration.seconds, pubsub.total.receivers from PubSubToBQ properties
+## Project that contains the output table
+pubsub.bigtable.output.project.id=<bigtable output project id>
```
Comment (vanshaj-bhatia) [inline java/src/main/resources/template.properties:157]: remove whatever if after =

Comment (balajiss2) [reply java/src/main/resources/template.properties:157]: Done

**Comment by vanshaj-bhatia** [comment]
LGTM!
Please test the template before merging and proceed with the next steps.

**Pull request merged**